app/views/ontologies/browser/browse.html.haml

```diff
@@ -77,7 +75,7 @@
         .browse-second-row
           .browse-search-bar
             .browse-search-container
-              %input{:name => "search", :placeholder => "Start typing to filter ontologies, e.g AGROVOC...", :type => "text", :value => @search, data: {action: "input->browse-filters#dispatchInputEvent"}}
+              %input{:name => "search", :placeholder => "Start typing to filter ontologies, e.g., AGROVOC...", :type => "text", :value => @search, data: {action: "input->browse-filters#dispatchInputEvent"}}
```

Comment:
if you can use the localized value, it would be better

Reply:
I couldn't find the localized value in the development branch, but i will try to update it. ;)

Reply:
Todo in an other iteration. @SirineMhedhbi